pages/common/vim.md

```diff
@@ -4,6 +4,8 @@
 > Pressing `i` enters edit mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
 > More information: <https://www.vim.org>.
 
+**Important:** If you accidentally open Vim and have never used it before, _do not_ press anything. Type `:q!` and hit <kbd>Enter</kbd> to quit.
```

Comment:
Normally, I would suggest that this is not part of the standard format for tldr pages, and should be removed. However, this is a very unusual page, and it adds value. HTML support though isn't in the [client spec](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md), so it'll have to be removed. (CommonMark support is in the spec though, hence why I support leaving this in).
```suggestion
**Important:** If you accidentally open Vim and have never used it before, _do not_ press anything. Type `:q!` and press enter to quit.
```

Hopefully we can prevent situations such as [this one](https://www.pixelstech.net/images/fun/normal/IMG-20170805-WA0000.jpg) :P ([source](https://www.pixelstech.net/fun/412-Exit-Vim))